service/rpc/das.go

```diff
@@ -0,0 +1,52 @@
+package rpc
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/celestiaorg/celestia-node/das"
+)
+
+const (
+	dasStateEndpoint = "/daser"
```

Comment:
`/daser/state` ?

Reply:
It would actually be nice to consolidate all of the current endpoints we have  / and also consolidate endpoints once we merge #802 . 

I will make an issue for this.

Reply:
Actually, the problem is, we have other endpoints like `/data_available`, `/namespaced_shares`, so I think it should be `/daser_state` maybe.

Reply:
Do we expect to have endpoints that will return other services state? Then we can have `/state/daser`, `/state/syncer`.

Reply:
+1 for daser_stare or daser/state